### PR TITLE
Fix814: Clow pointing model and station procedures

### PR DESCRIPTION
**Pull request by sergiopoppi**
Added pointing model and station procedure for the CCG configuration

### CODE REVIEW

File: SRT/CDB/alma/Procedures/StationProcedures/StationProcedures.xml [hunk 1]
```diff
@@ -20,11 +20,18 @@
 <Procedure name="setupCCG" args="0">
 <body>
 	receiversSetup=CCG
+	antennaSetup=CCG
 	chooseBackend=TotalPower
+	initialize=PPP
```
Comment (aorlati) [inline SRT/CDB/alma/Procedures/StationProcedures/StationProcedures.xml:25]: @sergio the "PPP" configuration of the focus selector sets the 730MHz filter cutting the receiver band. If this is not intended I suggest to add a CCG specific configuration

**Review by aorlati — COMMENTED** [review]
I just have a comment.

File: SRT/CDB/alma/Procedures/StationProcedures/StationProcedures.xml (hunk 1 above)
Comment (sergiopoppi) [reply SRT/CDB/alma/Procedures/StationProcedures/StationProcedures.xml:25]: @aorlati I do agree. Don't know how to do it.

Comment (sergiopoppi) [reply SRT/CDB/alma/Procedures/StationProcedures/StationProcedures.xml:25]: With setSection it's possibile to setup the choosen band also with PPP

Comment (aorlati) [reply SRT/CDB/alma/Procedures/StationProcedures/StationProcedures.xml:25]: Yes it's possible or you replace `initialize=CCG` and you add a 'CCG' configuration into this file: https://github.com/discos/discos/blob/master/SRT/Configuration/CDB/alma/DataBlock/TotalPower/TotalPower.xml
You can copy the PPP, change the configurationID and inputPortBw.

**Comment by giuseppe-carboni** [comment]
Closing this, I'll open a new PR with all the updates regarding the C-low receiver

**Pull request closed without merging**